scss/bitstyles/components/breadcrumbs.stories.mdx

```diff
@@ -0,0 +1,62 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
+
+<Meta title="Components/Breadcrumbs" />
+
+# Breadcrumbs
+
+A list of links charting the user’s place in the site structure. The title of the current page should be in an `<h1>` on the page, so it isn’t needed here.
+
+<Canvas isColumn>
+  <Story name="Breadcrumbs">
```

Comment:
For some reason I see this example jumping (even though there's no animation on it) 🤔 

![Kapture 2021-02-18 at 11 20 47](https://user-images.githubusercontent.com/3491815/108342768-7a62bd80-71db-11eb-9960-bf748fb9d2fd.gif)

Reply:
lolwut 😅 Thanks, I’ll look into it!

Reply:
Ok that’s weird  🙈It does the same with the second example if you view that one in the canvas mode then switch to docs. In Firefox it’s transitioned, in safari it’s not. I’ll put that down to a storybook bug for now, and reinvestigate later: https://github.com/bitcrowd/bitstyles/projects/1#card-55195339